lyaemu/spectra_mocking.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import fake_spectra
+from fake_spectra.plot_spectra import PlottingSpectra
+from fake_spectra.spectra import Spectra
+import matplotlib.pyplot as plt
+import array
+import h5py
+import seaborn as sn
+import os
+import subprocess
+
+def plot_pdf_Wiener_filtered(map_file=['60_512', '60_1024', '120_1024'],bins=np.arange(-1.0, 1.0, 0.02)):
```

Comment:
Docstring please. What is this function supposed to be doing? I can see what it does, but the docstring explains whether that is what you meant it to do.